app/javascript/utils/DelegationRequests.js

```diff
@@ -39,4 +40,18 @@ export async function delegateQuestions(answers, buildingId, email, firstName, l
     });
     addAnswers(answersToUpdate, buildingId);
   } catch (error) {}
+}
+
+export async function delegateBuildingQuestions(buildingId, userDetails, state, addAnswers, callbackFn=null) {
+  let answers = getAnswersForBuilding(buildingId, state);
+
+  await delegateQuestions(answers, buildingId, userDetails.email, userDetails.firstName, userDetails.lastName, addAnswers);
```

Comment:
I'm getting the same `delegations is empty` error from #239 when you try to assign a category that has already been assigned. Check out #264, we should add a `delegations.length == 0` check in `delegateQuestions`, or make the category disappear once assigned?

cc: @kevintli

Reply:
I think @didvi fixed this by graying out the "assign category" button if everything is already either answered or assigned - are you still able to do handoff when this is the case?

Reply:
I do think adding a check in `delegateQuestions` is good though just to prevent any future weird behavior! Divi, can you add that in before you merge this?